package.json

```diff
@@ -72,5 +73,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
If we start distinguishing between `deps` and `devDeps` here in the workspace root, all the existing `deps` should also be `devDeps` instead (as the workspace root ofc will never have actual 'production dependencies')

Reply:
It was a consequence of using `--dev`, I can change the others.